CONTRIBUTING.md

```diff
@@ -1,11 +1,21 @@
 Contributing
 ============
 
-Atramhasis is being developed as open source software by the 
+Atramhasis is an open source SKOS editor developed by
```

Comment:
The `the` is missing between `by` and `[Flanders...`

Reply:
It felt weird to have that the there, so I deleted it on purpose. But I'm not a native English speaker, so I might be wrong.

Reply:
Neither am I, I asked ChatGPT.

Reply:
I worked around the issue by dropping the agency bit. Think even ChatGPT should be satisfied now :-)